eppo/src/main/java/cloud/eppo/android/ConfigurationRequestor.java

```diff
@@ -29,7 +29,9 @@ public void load(InitializationCallback callback) {
             @Override
             public void onCacheLoadSuccess() {
                 cachedUsed.set(true);
-                callback.onCompleted();
+                if (callback != null) {
```

Comment:
The fix

(The [other two places](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/main/java/cloud/eppo/android/ConfigurationRequestor.java#L48) already have it)